static/sass/_snapcraft_custom-typography.scss

```diff
@@ -0,0 +1,43 @@
+@mixin snapcraft-custom-typography {
```

Comment:
Custom typography?

Reply:
These currently contain updates to line height for headings (multiline) used on home page and the max-width limiting based on font size (proposed and discussed on vanilla group before).

Reply:
Ok thanks, we need to fix the line-heights in Vanilla.